cmd/proxyplane/opv-proxyplane-http.example.json

```diff
@@ -0,0 +1,69 @@
+{
+  "version": 2,
+  "name": "OPV Proxy Plane",
+  "port": 28001,
+  "debug": false,
+  "host": [
+    "127.0.0.1"
+  ],
+  "cache_ttl": 3600,
+  "timeout": "3s",
+  "endpoints": [
+    {
+      "endpoint": "/tokenize",
+      "method": "POST",
+      "backend": [
+        {
+          "host": [
+            "https://httpbin.org"
+          ],
+          "url_pattern": "/post",
+          "extra_config": {
+            "github.com/devopsfaith/krakend-martian": {
+              "opv.body.Modifier": {
+                "scope": [
+                  "request"
```

Comment:
is this supposed to be an array?

Reply:
yes, the modifier can also work on `response`